cli/src/command/wallet.rs

```diff
@@ -53,6 +55,11 @@ pub enum WalletCommand {
     ChangePassword,
     /// Initialize the wallet.
     Init(InitParameters),
+    /// Migrate a stronghold v2 snapshot to v3.
+    MigrateStronghold {
+        /// Path of the to be migrated stronghold file. "./stardust-cli-wallet.stronghold" if nothing provided.
+        path: Option<String>,
+    },
```

Comment:
Is that only v2 -> v3. Is it fixed to this transition?

Reply:
Yes and hopefully never need another one